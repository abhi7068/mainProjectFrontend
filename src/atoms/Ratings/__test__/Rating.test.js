import React from 'react';

import Rating from '../index';
import { render } from '@testing-library/react';

it('it renders', () => {
  render(<Rating />);
});
it('should take a snapshot', () => {
  const { asFragment } = render(<Rating />);
  expect(asFragment(render(<Rating />))).toMatchSnapshot();
});
it('it renders with correct text', () => {
  const { getByTestId } = render(<Rating />);
  expect(getByTestId('Rating')).toHaveTextContent('Good');
});
