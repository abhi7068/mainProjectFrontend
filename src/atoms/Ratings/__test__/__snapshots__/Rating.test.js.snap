// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <p
      data-testid="Rating"
    >
      Good
    </p>
  </div>
</DocumentFragment>
`;
